packages/utils/src/types.ts

```diff
@@ -2,12 +2,6 @@ import type { Accessor, Setter } from "solid-js";
 
 export type { EffectOptions, OnOptions } from "solid-js";
 
-// export types that aren't exported by solid-js main module
-export type {
-  ResolvedJSXElement,
-  ResolvedChildren,
-} from "../node_modules/solid-js/types/reactive/signal.js";
```

Comment:
it could be breaking if someone imported the type from there accidentally. Yeah we we could do that, but how to ensure that we don't forget to clean that up on the next major?

Reply:
I usually just add a todo 🤷‍♂️

Reply:
Alright i will add the export again